templates/download/intel-nuc-desktop.html

```diff
@@ -1,90 +1,103 @@
-          <h3>Minimum requirements</h3>
-          <ul class="p-list">
-            <li class="p-list__item is-ticked">An Intel NUC with BIOS updated to the latest version (<a href="https://www.intel.com/content/www/us/en/support/articles/000033300/intel-nuc.html">update instructions</a>)</li>
-            <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (4GB minimum for Dawson Canyon NUCs, 2GB for older generations)</li>
-            <li class="p-list__item is-ticked">A USB keyboard and a mouse</li>
-            <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
-            <li class="p-list__item is-ticked">An HDMI cable</li>
-            <li class="p-list__item is-ticked">A network connection with Internet access</li>
-          </ul>
-        </div>
-      </div>
+    <div class="col">
+      <p>There are two install options for the Intel NUC: <a href="/download/iot/intel-nuc">Ubuntu Core</a> or <a href="/download/iot/intel-nuc-desktop#desktop">Ubuntu Desktop</a>. This page is for Ubuntu Desktop.</p>
+      <p>Use your favourite development tools, install snaps and access the vast Ubuntu ecosystem with your NUC.</p>
+      <hr class="p-rule" />
+      <h2 class="p-heading--5">Minimum requirements</h2>
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">An Intel&reg; NUC with BIOS updated to the latest version (<a href="https://www.intel.com/content/www/us/en/support/articles/000033300/intel-nuc.html">update instructions</a>)</li>
+        <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (4GB minimum for Dawson Canyon NUCs, 2GB for older generations)</li>
+        <li class="p-list__item is-ticked">A USB keyboard and a mouse</li>
+        <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
+        <li class="p-list__item is-ticked">An HDMI cable</li>
+        <li class="p-list__item is-ticked">A network connection with Internet access</li>
+      </ul>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-deep is-bordered">
-  <div class="u-fixed-width">
-    <h2>Installation instructions</h2>
+<section class="p-section">
+  <div class="row">
     <ol class="p-stepped-list--detailed">
       <li class="p-stepped-list__item">
-        <h3 class="p-stepped-list__title">
-
+        <hr class="p-rule u-hide--large" />
+        <h2 class="p-stepped-list__title">
           Download Ubuntu Desktop
-        </h3>
+        </h2>
         <div class="p-stepped-list__content">
           <p>Download the correct Ubuntu image for your device:</p>
-          <ul class="u-no-margin--left u-no-margin--top">
-            <li><a href="http://people.canonical.com/~platform/images/nuc/pc-dawson-xenial-amd64-m4-20180507-32.iso"> Dawson Canyon and June Canyon NUC &ndash; Ubuntu Desktop 16.04 LTS image</a>
+          <hr class="p-rule" />
+          <ul class="p-list--divided u-no-margin--left u-no-margin--top">
```

Comment:
```suggestion
          <ul class="p-list--divided">
```
These 2 aren't needed,  vanilla applies `margin-left: 0` to divided lists and `margin-top: 0` to uls natively